dotgov-websites/ocsp-crl.csv

```diff
@@ -49,7 +49,6 @@ ssee-ocss002.ocsp.dmz.pitc.gov
 399e-ocss001.ocsp.dmz.pitc.gov
 test1carepository.faa.gov
 niws_443_vs.nas.faa.gov
-www.nes.notams.faa.gov
```

Comment:
Why not put the new line here, so it replaces the erroneous one?

Reply:
I was not aware I could do so. I will keep this in mind if I come across this situation again. Do I need to cancel this PR and submit a new one?

Reply:
I took care of it.  See commit 7cc2a95.